stacks/root.yml

```diff
@@ -86,6 +90,8 @@ Mappings:
       prefix: 20
     Castle:
       prefix: 30
+    Feeder:
+      prefix: 32
```

Comment:
I would actually put Feeder as the top priority.  It gets a ton of traffic from Dovetail.  And apparently it costs us real life money for every rule a request is checked against.